src/app/market/buy/checkout-process/checkout-process.component.html

```diff
@@ -48,11 +48,13 @@
           <tr *ngFor="let listing of cart?.listings"
               [ngClass]="{expired: checkExpired(listing)}">
             <td class="item" fxLayout="row" fxLayoutGap="10px" fxLayoutAling="start center">
-              <div class="image" fxFlex="0 0 75px">
-                <img src="{{listing.imageCollection?.featuredImage?.thumbnail}}">
+              <div class="image" fxFlex="0 0 115px">
+                <img src="{{listing.imageCollection?.featuredImage?.thumbnail}}" (click)="openListing(listing)"
+                  matTooltip="Click for more info" matTooltipPosition="above">
               </div>
               <div class="info" fxFlex="1 1 100%" fxLayout="column" fxLayoutAlign="center start">
-                <div class="title">
+                <div class="title" (click)="openListing(listing)"
+                  matTooltip="Click for more info" matTooltipPosition="above">
```

Comment:
If the listing has been disabled due to an error message, then there is still the indication to the user to "click here for more information". Should such "help" indicators not be removed when the action indicated has been disabled/removed?